app/src/main/java/de/stefanmedack/ccctv/ui/events/EventsActivity.kt

```diff
@@ -35,11 +41,19 @@ class EventsActivity : BaseInjectableActivity() {
         fun start(activity: Activity, conference: Conference) {
             val intent = Intent(activity.baseContext, EventsActivity::class.java)
             intent.putExtra(CONFERENCE_ID, conference.id)
-            intent.putExtra(CONFERENCE_TITLE, conference.title)
+            intent.putExtra(EVENTS_VIEW_TITLE, conference.title)
             intent.putExtra(CONFERENCE_LOGO_URL, conference.logoUrl)
 
             activity.startActivity(intent, ActivityOptionsCompat.makeSceneTransitionAnimation(activity).toBundle())
         }
+
+        fun start(activity: Activity, searchQuery: String) {
```

Comment:
Since we would have two different ways to start this activity now, I would suggest to use more specific names for the start methods. E.g. `startForConference` and `startWithSearch` maybe?